cmd/pebble/main.go

```diff
@@ -336,14 +340,8 @@ func run() error {
 		fmt.Fprintf(Stderr, "WARNING: Cannot activate logging: %v\n", err)
 	}
 
-	pebbleDir := os.Getenv("PEBBLE")
-	if pebbleDir != "" && !osutil.IsDir(pebbleDir) {
-		return fmt.Errorf("$PEBBLE must point to a pebble directory")
-	}
-	clientConfig.Socket = os.Getenv("PEBBLE_SOCKET")
-	if clientConfig.Socket == "" {
-		clientConfig.Socket = filepath.Join(pebbleDir, ".pebble.socket")
-	}
+	_, socketPath := getEnvPaths()
```

Comment:
This would be fine:
_, clientConfig.Socket = getEnvPaths()

Reply:
Thanks, updated.